tests/python/unittest/test_auto_scheduler_feature.py

```diff
@@ -200,6 +201,35 @@ def test_gpu_feature():
         assert fequal(fea_dicts[0]["is_gpu"], 1.0)
 
 
+@T.prim_func
+def tir_matmul(
+    A: T.Buffer[(128, 128), "float32"],
+    B: T.Buffer[(128, 128), "float32"],
+    C: T.Buffer[(128, 128), "float32"],
+) -> None:
+    T.func_attr({"from_legacy_te_schedule": True, "global_symbol": "main", "tir.noalias": True})
+    for y in T.serial(128):
+        T.store(C.data, T.ramp(y, 128, 128), T.broadcast(T.float32(0), 128), T.broadcast(True, 128))
+        for k in T.serial(128):
+            T.store(
+                C.data,
+                T.ramp(y, 128, 128),
+                T.load("float32x128", C.data, T.ramp(y, 128, 128), T.broadcast(True, 128))
+                + T.load("float32x128", A.data, T.ramp(k, 128, 128), T.broadcast(True, 128))
+                * T.broadcast(T.load("float32", B.data, y * 128 + k), 128),
+                T.broadcast(True, 128),
+            )
```

Comment:
cc @Lunderberg @vinx13 @junrushao1994 on recommendation on using `T.load / store` in new tests.

Reply:
I rewrote this to use buffer store.

Reply:
:+1: